### DISCUSSION TITLE
Component not found

### DISCUSSION THREAD

**Discussion by johanpretoriusgithub** [adrianosantostreina/MobilePermissions#31] [General]
Create new Multi Device project in Delphi 11.2
Place MobilePermissions component onto the main form.
Do "Build Project".
Error =
[DCC Fatal Error] Unit1.pas(9): F2613 Unit 'MobilePermissions.Component' not found.

**Comment by adrianosantostreina** [comment]
This is not an error. Delphi needs to know where the component's .pas or .dcu are. Basic.
Add the component's path to the project's Search Path. It's written in the documentation.

**Reply by johanpretoriusgithub** [reply]
thank you
